extensions/applicationinsights-properties-js/Tests/Unit/src/SessionManager.Tests.ts

```diff
@@ -64,13 +64,12 @@ export class SessionManagerTests extends AITestClass {
             test: () => {
 
                 var sessionPrefix = newId();
-                var config = {
+                var config = createDynamicConfig({
```

Comment:
As the Session Manager now supports `onConfigChange` the passed in configuration needs to already be a dynamic configuration instance (which it will be already when initialized via the Core).

So for testing we need to simulate the same situation.